packages/react-core/src/demos/sampleData2.tsx

```diff
@@ -8,178 +7,153 @@ export interface SampleRowData {
 
 export const rows: SampleRowData[] = [
   {
-    name: `Node1`,
-    status: 'Running',
-    location: 'Raleigh',
-    lastModified: '1 hour ago',
+    status: 'running',
+    location: 'raleigh',
+    lastModified: 'oneHr',
     url: 'http://www.redhat.com/en/office-locations/node1'
```

Comment:
Really more a nit since it looks to work fine for this demo's purpose, but we could take out the url property from this data file and dynamically create the string in the generateRowsFromTranslation function similar to the `name` property there (appending the `index` number at the end of the base url string).